packages/common/src/components/CondaPkgList.tsx

```diff
@@ -143,6 +147,23 @@ export class CondaPkgList extends React.Component<IPkgListProps> {
       return <span>{rowData.name}</span>;
     };
 
+    const versionRender = ({ rowData }: ICellRender): JSX.Element => (
+      <a
+        className={
+          rowData.updatable
+            ? classes(Style.Updatable, Style.Link)
+            : Style.Link
+        }
+        href="#"
+        onClick={(evt): void => {
+          this.props.onPkgGraph(rowData);
```

Comment:
You need to stop the propagation of the event otherwise it will trigger an update or a remove action on the package.

```suggestion
          evt.stopPropagation();
          this.props.onPkgGraph(rowData);
```